src/Semantic/Api/V1/CodeAnalysisPB.hs

```diff
@@ -1,5 +1,5 @@
 -- Code generated by protoc-gen-haskell 0.1.0, DO NOT EDIT.
-{-# LANGUAGE DerivingVia, DeriveAnyClass, DuplicateRecordFields #-}
+{-# LANGUAGE DerivingVia, DeriveAnyClass, DuplicateRecordFields, PatternSynonyms #-}
```

Comment:
Does the above comment imply that our pattern synonyms will be overwritten?

Reply:
Yes, unfortunately (though we haven't regenerated this file in ages). I'm going to file a bug with the upstream proto3-suite to add generation of pattern synonyms for `oneof` types.